tests/fixtures/require-spread/good/super.js

```diff
@@ -4,6 +4,7 @@ let Blah = Foo.extend({
 
     // odd, but still needed in some cases
     let { _super } = this;
+
```

Comment:
I think we should remove the `newline-after-var` rule. I'm not sure how it made its way into the config in the first place, but I'd rather not introduce any new styling rule without discussing with the team first. And my personal opinion is that something like the following should be accepted since it's such a short block:

```js
function greeting(id) {
  let name = generateName(id);
  return `Hola, ${name}!`;
}
```